src/diet.ts

```diff
@@ -668,19 +668,31 @@ export function computeDiet(): {
   shotglass: () => Diet<Note>;
   pantsgiving: () => Diet<Note>;
   sweatpants: () => Diet<Note>;
+  stooper: () => Diet<Note>;
+  nightcap: () => Diet<Note>;
 } {
   // Handle spleen manually, as the diet planner doesn't support synth. Only fill food and booze.
 
   const orEmpty = (diet: Diet<Note>) =>
     diet.expectedValue(MPA, "net") < 0 ? new Diet<Note>() : diet;
+
+  const DRUNK_FACTOR = 0.75;
```

Comment:
nice

Reply:
Do we want to allow the user to self-define Drunk_Factor (whether via relay, setting a preference or giving an argument when running?

Reply:
.75 seems a little low but fine as an initial estimate, I probably lose something like 1000/6000 MPA when overdrunk